bolt/test/X86/jt-symbol-disambiguation-4.s

```diff
@@ -0,0 +1,60 @@
+# If the operand references a symbol that differs from the jump table label,
+# no reference updating is required even if its target address resides within
+# the jump table's range.
+# In this test case, consider the second instruction within the main function,
+# where the address resulting from 'c + 17' corresponds to one byte beyond the
+# address of the .LJTI2_0 jump table label. However, this operand represents
+# an offset calculation related to the global variable 'c' and should remain
+# unaffected by the jump table.
+
+# REQUIRES: system-linux
+
+
+# RUN: %clang -no-pie  %s -o %t.exe -Wl,-q
+
+# RUN: %t.exe
```

Comment:
Runnable tests need to be under bolt/test/runtime. But in this case, there's no need to run the test to verify the behavior. Please remove these lines with running the binary.